### PR TITLE
fix: Minor spelling typos

### PR DESCRIPTION
**Pull request by ottok**
Fix various minor spelling mistakes.

Typo 'SEPERATOR' (SEPARATOR) and 'collapsable' (collapsible) were not fixed as it will require more testing to ensure there are no side effects.

## Summary by CodeRabbit

- **文档更新**
	- 修正了多个文件中的拼写和语法错误
	- 改进了注释和文档的准确性
	- 更新了一些按钮和标签的文本标签

- **代码质量**
	- 对多个组件和模块的注释进行了细微的文字修正
	- 未对代码功能和逻辑进行实质性更改

这些更改主要集中在提高代码的可读性和文档的准确性，不会影响系统的核心功能。

### CODE REVIEW

**Comment by ottok** [comment]
Note to self: The text in Chinese above basically says multiple times:

> Overview of Code Changes
> This pull request mainly contains spelling and grammar corrections in multiple files, including comments, error messages, and user interface text. These changes are intended to improve the readability and accuracy of the code, but have no substantial impact on the functionality or logic of the code.

File: packages/core/src/chains/xmr/sdkXmr/moneroCore/moneroCore.js [hunk 1]
```diff
@@ -3336,7 +3336,7 @@ var MyMoneroLibAppCpp = (() => {
         // parents must exist
         var lookup, old_dir, new_dir;
 
-        // let the errors from non existant directories percolate up
+        // let the errors from non existent directories percolate up
         lookup = FS.lookupPath(old_path, { parent: true });
         old_dir = lookup.node;
```
[the remark this answers is not in the document]
Comment (ottok) [reply packages/core/src/chains/xmr/sdkXmr/moneroCore/moneroCore.js:3341]: Out of scope

File: packages/core/src/chains/xmr/sdkXmr/moneroCore/moneroCore.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ var MyMoneroLibAppCpp = (() => {
     // Normally we don't log exceptions but instead let them bubble out the top
     // level where the embedding environment (e.g. the browser) can handle
     // them.
-    // However under v8 and node we sometimes exit the process direcly in which case
+    // However under v8 and node we sometimes exit the process directly in which case
     // its up to use us to log the exception before exiting.
     // If we fix https://github.com/emscripten-core/emscripten/issues/15080
```
[the remark this answers is not in the document]
Comment (ottok) [reply packages/core/src/chains/xmr/sdkXmr/moneroCore/moneroCore.js]: Updated accordingly

**Review by originalix — APPROVED** [review]

**Comment by ottok** [comment]
It is unclear if monero.js is actually your own code, or if the improvements should be sent upstream somewhere.

In general, the project README.md is very thin, and it is hard for an outsider to reason about the code architecture, so these changes are so far minimal.

**Review by sidmorizon — APPROVED** [review]

**Review by ezailWang — APPROVED** [review]

**Review by revan-zhang — APPROVED** [review]

**Pull request merged**